dotcom-rendering/src/web/components/ElementContainer.tsx

```diff
@@ -58,6 +59,7 @@ export const ElementContainer = ({
 	shouldCenter = true,
 	children,
 	element = 'div',
+	className = null,
```

Comment:
if the classname is optional, we shouldn't need to set it here

Reply:
nice, will omit